run-spill-build/overlaySinglesIntoSpillsSorted.C

```diff
@@ -5,6 +5,14 @@
 // containing single neutrino interactions (from some flux) in the
 // event tree and overlay these events into full spills.
 //
+// The macro can build LAr only spills (setting inFile2POT to 0),
+// rock only spills (setting inFile1POT to 0) or LAr + rock spills
+// (setting both inFile1POT and inFile2POT to be greater than 0).
+//
+// When building LAr only spills one can run in "N Interaction" mode
+// by specifying a number less than or equal to 100 as spillPOT. Useful,
```

Comment:
10000, not 100?

Reply:
Good catch - this number changed in response to a comment from Kevin above. I update the variable but not the comment 🙏 fixed in [339348f](https://github.com/DUNE/2x2_sim/pull/42/commits/339348f2252266d51be93bd18327859175ef51c3)